mlflow/models/dependencies_schemas.py

```diff
@@ -34,12 +37,12 @@ def set_retriever_schema(
     determine which fields correspond to the document text, document URI, etc.
 
     Args:
```

Comment:
Rename this to Kwargs?

Or basically I have seen in Mlflow so we might follow that pattter?
```
Args:
    kwargs:
        ....
```

Reply:
I think Kwargs should be fine? As we're not having a `kwargs` named `Arg`

Reply:
but these are not kwargs